src/index.js

```diff
@@ -0,0 +1,174 @@
+//--API--//
+
+function getForecast(coordinates) {
+    console.log(coordinates);
+    let apiKey = "f2a7bf87b777299f2d3968c89592e347";
```

Comment:
don't put API key in the code, people can copy it and use and if you by accident pay for the API, they can charge you for usage. There are other ways deploying your code with API key, that doesn't have to be in the code directly